### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by quentinleroy**

### CODE REVIEW

File: source/includes/_bookings.md [hunk 1]
```diff
@@ -948,10 +1101,10 @@ Error Code  | Meaning
...
 ```
 
-Affiliates that do not use our [bookings#create](https://api-demo.distribusion.com/retailers/v4/docs/#create) endpoint, can redirect their customers to our booking page after searching connections on [connections#find](https://api-demo.distribusion.com/retailers/v4/docs/#find). The locale will be taken from the setting of the browser, which the customer is using. The currency can be set within the redirect link. You can use the following
+Affiliates that do not use our [bookings#create](https://api-demo.distribusion.com/retailers/v4/docs/#create) endpoint, can redirect their customers to our booking page after searching connections on [connections#find](https://api-demo.distribusion.com/retailers/v4/docs/#find). The locale will be taken from the setting of the browser, which the customer is using. Alternatively, you can overwrite this functionality by sending your own locale. The currency can be set within the redirect link.
```
Comment (shudayo) [inline source/includes/_bookings.md]: Priority now is:
1. Locale param in URL
2. cached locale
3. browser locale

Maybe we want to specify it there?

File: source/includes/_introduction.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Distribusion aims to standardize the station codes across the bus industry, the
...
 
-Currently our content can be fetch in English, French, Italian and German, with the following locale values: en, fr, it and de.
+Currently our content can be fetch in English, French, Italian, Spanish, Bulgarian and German, with the following locale values: en, fr, it, es, bg and de.
```
Comment (shudayo) [inline source/includes/_introduction.md]: What about dutch and swedish?

Comment (quentinleroy) [reply source/includes/_introduction.md]: Very limited content translated - but will add them.

File: source/includes/_bookings.md (hunk 1 above)
Comment (quentinleroy) [reply source/includes/_bookings.md]: Thanks for the input, will add this along.

**Pull request merged**